File: swap.py
import sys

with open('d:/Portfolio/index.html', 'r', encoding='utf-8') as f:
    lines = f.readlines()

def find_idx(lines, marker):
    for i, l in enumerate(lines):
        if marker in l: return i
    return -1

ed_start = find_idx(lines, '<!-- Education Section -->')
sk_start = find_idx(lines, '<!-- Skills Section -->')
pr_start = find_idx(lines, '<!-- Projects Section -->')

if ed_start != -1 and sk_start != -1 and pr_start != -1:
    ed_section = lines[ed_start:sk_start]
    sk_section = lines[sk_start:pr_start]
    
    for i, l in enumerate(ed_section):
        if 'id=\"education\"' in l:
            ed_section[i] = l.replace(' alt-bg', '')
            break
            
    for i, l in enumerate(sk_section):
        if 'id=\"skills\"' in l:
            sk_section[i] = l.replace('class=\"skills section\"', 'class=\"skills section alt-bg\"')
            break
            
    new_lines = lines[:ed_start] + sk_section + ed_section + lines[pr_start:]
    with open('d:/Portfolio/index.html', 'w', encoding='utf-8') as f:
        f.writelines(new_lines)
    print("Swapped sections successfully!")
else:
    print("Could not find sections.")

File: swap.js
const fs = require('fs');
let html = fs.readFileSync('d:/Portfolio/index.html', 'utf8');

const edStart = html.indexOf('    <!-- Education Section -->');
const skStart = html.indexOf('    <!-- Skills Section -->');
const prStart = html.indexOf('    <!-- Projects Section -->');

if (edStart > -1 && skStart > -1 && prStart > -1) {
    let edSec = html.substring(edStart, skStart);
    let skSec = html.substring(skStart, prStart);
    
    edSec = edSec.replace('class=\"education section alt-bg\"', 'class=\"education section\"');
    skSec = skSec.replace('class=\"skills section\"', 'class=\"skills section alt-bg\"');
    
    html = html.substring(0, edStart) + skSec + edSec + html.substring(prStart);
    fs.writeFileSync('d:/Portfolio/index.html', html);
    console.log('Swapped!');
} else {
    console.log('Not found');
}
